app/models/status_presenter_factory.rb

```diff
@@ -6,7 +6,7 @@ def initialize(auction)
   end
 
   def create
-    "StatusPresenter::#{status_name}".constantize.new(auction)
+    Object.const_get("StatusPresenter::#{status_name}").new(auction)
```

Comment:
no, just trying to make this more consistent with https://github.com/18F/micropurchase/blob/develop/app/models/sam_status_presenter_factory.rb#L9